### PR TITLE
server: change MAX_GRPC_SEND_MSG_LEN

### PR DESCRIPTION
**Pull request by choleraehyq**
Set region size to 256MB may cause grpc msg length bigger than 128M.

@zhangjinpeng1987 @disksing PTAL

Signed-off-by: Cholerae Hu <huyingqian@pingcap.com>

### CODE REVIEW

File: src/server/server.rs [hunk 1]
```diff
@@ -33,7 +33,7 @@ use super::raft_client::RaftClient;
 
 const DEFAULT_COPROCESSOR_BATCH: usize = 50;
 const MAX_GRPC_RECV_MSG_LEN: usize = 10 * 1024 * 1024;
-const MAX_GRPC_SEND_MSG_LEN: usize = 128 * 1024 * 1024;
+const MAX_GRPC_SEND_MSG_LEN: usize = 8 * 1024 * 1024 * 1024;
```
Comment (BusyJay) [inline src/server/server.rs]: Why not update it according to the size of region?

File: src/server/server.rs [hunk 2]
```diff
@@ -33,7 +34,7 @@ use super::raft_client::RaftClient;
 
 const DEFAULT_COPROCESSOR_BATCH: usize = 50;
 const MAX_GRPC_RECV_MSG_LEN: usize = 10 * 1024 * 1024;
-const MAX_GRPC_SEND_MSG_LEN: usize = 128 * 1024 * 1024;
+const MAX_GRPC_SEND_MSG_LEN: usize = REGION_SPLIT_SIZE as usize * 1024 * 1024;
```
Comment (zhangjinpeng1987) [inline src/server/server.rs]: How about 128 or 64 * REGION_SPLIT_SIZE?

File: src/server/server.rs [hunk 3]
```diff
@@ -33,7 +34,7 @@ use super::raft_client::RaftClient;
 
 const DEFAULT_COPROCESSOR_BATCH: usize = 50;
 const MAX_GRPC_RECV_MSG_LEN: usize = 10 * 1024 * 1024;
-const MAX_GRPC_SEND_MSG_LEN: usize = 128 * 1024 * 1024;
+const MAX_GRPC_SEND_MSG_LEN: usize = 128 * REGION_SPLIT_SIZE as usize * 1024 * 1024;
```
Comment (siddontang) [inline src/server/server.rs]: `REGION_SPLIT_SIZE` is already `256 * 1024 * 1024`. I think you use a huge length here.

Comment (choleraehyq) [inline src/server/server.rs]: Yes. It has no side effect.

Comment (siddontang) [inline src/server/server.rs]: If you want to use `REGION_SPLIT_SIZE`, use a small multiplier, not `128 * 1024 * 1024`, the bad value here.

Or use a fixed value directly, like 1 GB, 4 GB or 8 GB.

Comment (choleraehyq) [inline src/server/server.rs]: Yeah, my fault.

File: src/server/server.rs [hunk 4]
```diff
@@ -33,7 +34,7 @@ use super::raft_client::RaftClient;
 
 const DEFAULT_COPROCESSOR_BATCH: usize = 50;
 const MAX_GRPC_RECV_MSG_LEN: usize = 10 * 1024 * 1024;
-const MAX_GRPC_SEND_MSG_LEN: usize = 128 * 1024 * 1024;
+const MAX_GRPC_SEND_MSG_LEN: usize = 128 * REGION_SPLIT_SIZE as usize;
```
Comment (BusyJay) [inline src/server/server.rs]: I think `1.5~2` should be enough.

**Comment by choleraehyq** [comment]
@BusyJay Updated to 4x.

**Comment by BusyJay** [comment]
LGTM

**Pull request merged**